recipes/hdf5-plugins/bld.bat

```diff
@@ -0,0 +1,4 @@
+mkdir build
+cd build
+cmake .. -DENABLE_BITSHUFFLE_PLUGIN=yes -DENABLE_LZ4_PLUGIN=yes -DENABLE_BZIP2_PLUGIN=yes -DCMAKE_INSTALL_PREFIX=$PREFIX -G"Visual Studio %VS_MAJOR% %VS_YEAR% Win64"
```

Comment:
I think you need to check the return codes after each command in batch scripts?

Reply:
done